packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -377,8 +377,8 @@ export class HttpPlugin extends BasePlugin {
           const tags = new TagMap();
           tags.set(stats.HTTP_CLIENT_METHOD, {value: method});
 
-          if (options.hostname) {
```

Comment:
I think ```hostname``` is preferred over ```host```. I would suggest to add below line.
```const host = options.hostname || options.host || 'localhost';``` WDYT?

Reply:
```
const host = options.hostname || options.host || 'localhost';
span.addAttribute(HttpPlugin.ATTRIBUTE_HTTP_HOST, host);
```
This change will also fix failing build.

Reply:
thanks!